src/pages/questions/ui/QuestionsPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { NavLink, Navigate } from 'react-router-dom';
+import cl from './QuestionsPage.module.scss';
+import PhoneImg from 'shared/assets/img/phone.svg';
+import { Mark } from 'widgets/mark';
+import { Button } from 'shared/ui/button/Button';
+import { useAppSelector } from 'shared/lib/hooks/useAppSelector/useAppSelector';
+import { useAppDispatch } from 'shared/lib/hooks/useAppDispatch/useAppDispatch';
+import { fetchQuestions } from 'entities/questions';
+import { getCompletedSurvey, getMain, getResult } from 'app/providers/router/routeConfig/routes';
+import { useQuestionsCompleted } from 'shared/lib/hooks/useQuestionsCompited/useQuestionsCompited';
+
+const QuestionsPage: React.FC = () => {
```

Comment:
обрати внимание, чтобы во всех компонентах у тебя была одинаковая типизация
например
```suggestion
const QuestionsPage: React.FC = (): JSX.Element => {
```

Reply:
не во всех компонентах у тебя это есть